kuma/wiki/models.py

```diff
@@ -1492,6 +1493,10 @@ def zone_stack(self):
     def get_full_url(self):
         return absolutify(self.get_absolute_url())
 
+    @property
+    def is_experiment(self):
+        return self.slug.startswith(EXPERIMENT_TITLE_PREFIX)
```

Comment:
Ah, okay, so I was doing it wrong. We're assuming all experiments will be at the `/docs/` root (and most likely not following the "control" article's URL structure).